locales/en-US/app.ftl

```diff
@@ -150,6 +150,14 @@ CallTree--bytes-self = Self (bytes)
         The “self” bytes are useful for understanding where memory was actually
         allocated or deallocated in the program.
 
+# These inlining badges are displayed in the call tree in front of some
```

Comment:
I assume this applies to both strings. If so, it should be a group comment (two hashes, leave an empty line after).

Please always add comment for variables. You can see examples of the format around line 90.

Reply:
Ah, thanks, will do!